images/runtime/php-fpm/8.3/docker-php-entrypoint

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+# first arg is `-f` or `--some-option`
+if [ "${1#-}" != "$1" ]; then
+	set -- php-fpm "$@"
+fi
+
+exec "$@"
```

Comment:
These files look redundant, let's keep them at a common location for future

Reply:
There are a set of files which are common with every version like 8.2, 8.1, 8.0..
To keep it at common place we have make some changes in dockerfile generator script.
And because of limited understanding of the system.. keeping it same like others.